api_tests/utils.py

```diff
@@ -14,4 +14,4 @@ def create_test_file(node, user):
         'size': 1337,
         'contentType': 'img/png'
     }).save()
-    return test_file
+    return test_file
```

Comment:
Did you run Flake8 on this before you pushed? This looks like it might fail on Travis.